.ci-builds/.azure-pipelines.yml

```diff
@@ -628,45 +628,52 @@ jobs:
+        mkdir -p icuexportdata/norm/small
+        ./bin/icuexportdata --mode norm --index --copyright --verbose --destdir icuexportdata/norm/small --trie-type small --all
+      displayName: 'Build normalization data export files'
+      env:
+        LD_LIBRARY_PATH: lib
     - script: |
-        mkdir -p icu4c/source/icuexportdata_uprops_full/collation_unihan
-        mkdir -p icu4c/source/icuexportdata_uprops_full/collation_implicithan
         cd icu4c/source
-        cd data/coll
-        FILES=`echo *.txt`
-        cd -
-        LD_LIBRARY_PATH=lib ./bin/genrb -X -s data/coll/ -d icuexportdata_uprops_full/collation_unihan --ucadata data/in/coll/ucadata-unihan-icu4x.icu $FILES
-        LD_LIBRARY_PATH=lib ./bin/genrb -X -s data/coll/ -d icuexportdata_uprops_full/collation_implicithan --ucadata data/in/coll/ucadata-implicithan-icu4x.icu $FILES
-        rm icuexportdata_uprops_full/collation_unihan/*.res
-        rm icuexportdata_uprops_full/collation_implicithan/*.res
-      displayName: 'Build collation data export file'
-    - task: PublishBuildArtifacts@1
-      displayName: 'Publish Artifact: icuexportdata_uprops_full'
+        mkdir -p icuexportdata/ucase/fast
+        ./bin/icuexportdata --mode ucase --index --copyright --verbose --destdir icuexportdata/ucase/fast --trie-type fast --all
+        mkdir -p icuexportdata/ucase/small
+        ./bin/icuexportdata --mode ucase --index --copyright --verbose --destdir icuexportdata/ucase/small --trie-type small --all
+      displayName: 'Build case data export files'
+      env:
+        LD_LIBRARY_PATH: lib
+    - script: |
+        cd icu4c/source
+        FILES=`ls data/coll "*.txt"` 
+        mkdir -p icuexportdata/collation/unihan
+        ./bin/genrb -X -s data/coll/ --ucadata data/in/coll/ucadata-unihan-icu4x.icu -d icuexportdata/collation/unihan $FILES
+        rm icuexportdata/collation/unihan/*.res
+        mkdir -p icuexportdata/collation/implicithan
+        ./bin/genrb -X -s data/coll/ --ucadata data/in/coll/ucadata-implicithan-icu4x.icu -d icuexportdata/collation/implicithan $FILES
+        rm icuexportdata/collation/implicithan/*.res
+      displayName: 'Build collation data export files'
+      env:
+        LD_LIBRARY_PATH: lib
+    - task: ArchiveFiles@2
+      displayName: 'Zip'
       inputs:
-        PathtoPublish: 'icu4c/source/icuexportdata_uprops_full'
-        ArtifactName: 'icuexportdata_uprops_full'
+        rootFolderOrFile: 'icu4c/source/icuexportdata' 
+        includeRootFolder: false 
+        archiveFile: 'icuexportdata_version-goes-here.zip' 
     - task: PublishBuildArtifacts@1
-      displayName: 'Publish Artifact: icuexportdata_uprops_sample'
+      displayName: 'Publish Artifact'
       inputs:
-        PathtoPublish: 'icu4c/source/icuexportdata_uprops_sample'
-        ArtifactName: 'icuexportdata_uprops_sample'
+        PathtoPublish: 'icuexportdata_version-goes-here.zip'
+        ArtifactName: 'output'
```

Comment:
Issue: `output.zip` is quite generic; see https://dev.azure.com/ms/icu/_build/results?buildId=325853&view=artifacts&type=publishedArtifacts